lab2.js

```diff
@@ -122,10 +153,17 @@ assert(human.sayHello(klingon) === 'nuqneH',
 //*********************************************************
 function max(array) {
   // TODO: return the largest number in the given array
+  var max = Math.max([1, 3, 2]);
+  var max2 = Math.max([3, 54, 65]);
+  var max3 = Math.max([34, 5445, 332]);
+  var max4 = Math.max([2, 4, 6]);
 }
 
 // TODO: write three more assertions
 assert(max([ 1, 3, 2 ]) === 3, '[1,3,2]');
+assert(max2([3, 54, 65]) === 65, '[3,54,65]');
```

Comment:
this is where your asserts aren't passing...it's expecting max2 to be a function. You should just be able to call max with a given array and then that return value is what gets compared to 65.